compositor_render/src/wgpu/format/pixel_yuv_to_rgba.wgsl

```diff
@@ -0,0 +1,52 @@
+struct VertexInput {
+    @location(0) position: vec3<f32>,
+    @location(1) tex_coords: vec2<f32>,
+}
+
+struct VertexOutput {
+    @builtin(position) position: vec4<f32>,
+    @location(0) tex_coords: vec2<f32>,
+}
+
+@vertex
+fn vs_main(input: VertexInput) -> VertexOutput {
+    var output: VertexOutput;
+
+    output.position = vec4(input.position, 1.0);
+    output.tex_coords = input.tex_coords;
+
+    return output;
+}
+
+@group(0) @binding(0) var texture: texture_2d<f32>;
+@group(1) @binding(0) var sampler_: sampler;
+
+@fragment
+fn fs_main(input: VertexOutput) -> @location(0) vec4<f32> {
+    var dimensions = textureDimensions(texture);
+    var eps = 0.0001;
+    var half_pixel_width = 0.5 / f32(dimensions.x);
+
+    // x_pos represents index of a column(pixel) on the output texture
+    // - dimensions.x represents half of the output width, so we need multiply * 2
```

Comment:
Only for u and v planes. This description can be misleading.

Reply:
there are no u and v planes this is not a planar format it refers to width of texture in pixels

Reply:
Ok, I don't know how to formulate this, but the "Y component" has the same resolution. Only "U and V components" have reduced width.

Reply:
Input texture if in RGBA format (but each component is redefined).  Single "pixel" in the input texture has 4 bytes which includes two Y "value", So width of texture is half of the output one when talking about pixels.

Reply:
Ok, at first, I didn't get that. Now it makes sense.